Client/game_sa/CStreamingSA.cpp

```diff
@@ -152,3 +152,8 @@ void CStreamingSA::ReinitStreaming()
 
     reinitStreaming();
 }
+
+void CStreamingSA::MakeSpaceFor(std::uint32_t memoryToCleanInBytes)
+{
+    ((int(__cdecl*)(std::uint32_t))0x40E560)(memoryToCleanInBytes);
```

Comment:
```suggestion
    using FunctionSignature = void(__cdecl*)(std::uint32_t);
    auto Function = reinterpret_cast<FunctionSignature>(0x40E120);
    Function(memoryToCleanInBytes);
```
`0x40E560` is the address of `static void __cdecl CStreaming::ReInit(void)`

Reply:
```suggestion
    ((int(__cdecl*)(std::uint32_t))0x40E120)(memoryToCleanInBytes);
```